tests_integration_with_local_chain/markets/omen/test_stealing_on_markets.py

```
@@ -0,0 +1,214 @@
+    # Buying or selling tokens after the liquidity is removed will fail.
+    with pytest.raises(Exception) as e_buying:
+        binary_omen_buy_outcome_tx(
+            api_keys_B,
+            buy_yes_for_b,
+            agent_market,
+            binary_outcome=True,
+            auto_deposit=True,
+            web3=local_web3,
+        )
+    sell_yes_for_b = xdai_type(1)
+    with pytest.raises(Exception) as e_selling:
```

Ruff

Local variable `e_selling` is assigned to but never used

Remove assignment to unused variable `e_selling`

(F841)